raft/src/main/java/org/apache/kafka/raft/RaftConfig.java

```diff
@@ -37,49 +36,56 @@
     private static final String QUORUM_PREFIX = "quorum.";
 
     public static final String QUORUM_VOTERS_CONFIG = QUORUM_PREFIX + "voters";
-    private static final String QUORUM_VOTERS_DOC = "Map of id/endpoint information for " +
+    public static final String QUORUM_VOTERS_DOC = "Map of id/endpoint information for " +
         "the set of voters in a comma-separated list of `{id}@{host}:{port}` entries. " +
         "For example: `1@localhost:9092,2@localhost:9093,3@localhost:9094`";
+    public static final String QUORUM_VOTERS_DEFAULT = "";
```

Comment:
The typical approach used by other config files is to have `DEFAULT` as a prefix. See `ConsumerConfig` for example. Unless there's a reason to deviate, I'd follow the same approach.

Reply:
Makes sense. No reason to keep it this way. Fixed. Thanks!